activemerchant.gemspec

```diff
@@ -29,6 +29,7 @@ Gem::Specification.new do |s|
 
   s.add_development_dependency('mocha', '~> 1')
   s.add_development_dependency('pry')
+  s.add_development_dependency('pry-byebug')
```

Comment:
does the lock file not need to be updated?

Reply:
It is not needed since it is defined in the `gemspec` - also, there is not `lock` file in AM.

Reply:
I did not notice that the `Gemfile.lock` was not in GH, just local. Tx so much!